shuup/simple_supplier/notify_events.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.dispatch import receiver
+
+from shuup.core.signals import shipment_created
+from shuup.notify import Event, Variable
+from shuup.notify.typology import Model
+
+from .models import StockCount
+from .signals import stock_updated
+
+
+class AlertLimitReached(Event):
+    identifier = "alert_limit_reached"
+
+    supplier = Variable("Supplier", type=Model("shuup.Supplier"))
+    product = Variable("Product", type=Model("shuup.Product"))
+
+
+def _check_stock_level(supplier, product):
+    sc, _ = StockCount.objects.get_or_create(supplier=supplier, product=product)
+    if sc.physical_count < sc.alert_limit:
+        AlertLimitReached(supplier=supplier, product=product).run()
+
+
+@receiver(shipment_created)
+def check_order_stock_levels(order, **kwargs):
```

Comment:
This might not be needed either since shipment save updates the stocks and also the alert limit should be handled with that stock update.